dir.targets

```diff
@@ -57,6 +57,12 @@
       <DestinationDir>test-runtime/</DestinationDir>
     </SupplementalTestData>
   </ItemGroup>
+  
+  <PropertyGroup>
```

Comment:
I'm fine with this being here for now but once we get this all working we should put this in test.targets in build tools. .